docs/content/installation/helm.md

```diff
@@ -90,13 +90,26 @@ PGO will check for updates upon startup and once every 24 hours. Any errors in c
 
 ## Upgrade and Uninstall
 
-And once PGO has been installed, it can then be upgraded and uninstalled using applicable `helm`
-commands:
+Once PGO has been installed, it can then be upgraded using the `helm upgrade` command.
+However, before running the `upgrade` command, any Custom Resource Definitions (CRDs) must first be
+manually updated (this is specifically due to a
+[design decision in Helm v3](https://helm.sh/docs/chart_best_practices/custom_resource_definitions/),
+in which any CRDs in the Helm chart are only applied when using the `helm install` command).
+
+Therefore, to first update the PGO CRDs, run the following command:
+
+```shell
+kubectl apply -n <namespace> -f helm/install/crds
```

Comment:
🔧 CRDs have no namespace.

```suggestion
kubectl apply -f helm/install/crds
```

<br/>

🤔 I want to suggest `--server-side` so the last-applied-configuration annotation doesn't get too big, but I encountered conflicts with helm:

```console
$ kubectl apply --server-side -f helm/install/crds
error: Apply failed with 3 conflicts: conflicts with "helm" using apiextensions.k8s.io/v1:
- .metadata.annotations.controller-gen.kubebuilder.io/version
- .metadata.labels.app.kubernetes.io/version
- .spec.versions

Please review the fields above--they currently have other managers. Here
are the ways you can resolve this warning:
* If you intend to manage all of these fields, please re-run the apply
  command with the `--force-conflicts` flag.
* If you do not intend to manage all of the fields, please edit your
  manifest to remove references to the fields that should keep their
  current managers.
* You may co-own fields by updating your manifest to match the existing
  value; in this case, you'll become the manager if the other manager(s)
  stop managing the field (remove it from their configuration).
See https://kubernetes.io/docs/reference/using-api/server-side-apply/#conflicts
```